docs/core/deploying/index.md

```diff
@@ -1,95 +1,120 @@
+- **More space required across all apps**\
+Each copy of a self-contained app includes its own copy of the .NET Core files, more disk space is consumed.
 
-It also has a number of disadvantages:
+## Runtime-dependent
 
-- Because .NET Core is included in your deployment package, you must select the target platforms for which you build deployment packages in advance.
+Runtime-dependent apps are cross-platform and don't include the .NET Core runtime, but require the user of your app to install the .NET Core runtime.
 
-- The size of your deployment package is relatively large, since you have to include .NET Core as well as your app and its third-party dependencies.
+A *dll* file is created when you publish your app. For example, if you have an app named **word_reader**, a file named *word_reader.dll* is created. Apps built in this way are run with the `dotnet file.dll` command and can be run on any platform.
 
-  Starting with .NET Core 2.0, you can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core [*globalization invariant mode*](https://github.com/dotnet/runtime/blob/master/docs/design/features/globalization-invariant-mode.md). Ordinarily, .NET Core on Linux relies on the [ICU libraries](http://icu-project.org) for globalization support. In invariant mode, the libraries are not included with your deployment, and all cultures behave like the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture?displayProperty=nameWithType).
+Along with the *dll* file containing your app, a platform-specific executable can be published if the `-r <RID>` parameter is used. The *dll* is still created and remains cross-platform while the executable isn't. Continuing the example above, if you publish the **word_reader** app and target Windows, a *word_reader.exe* executable is created along with *word_reader.dll*. When targeting Linux or macOS, a *word_reader* executable is created along with *word_reader.dll*. For more information about RIDs, see [.NET Core RID Catalog](../rid-catalog.md).
 
-- Deploying numerous self-contained .NET Core apps to a system can consume significant amounts of disk space, since each app duplicates .NET Core files.
+Starting with .NET Core 3.0 SDK, an executable is always created for your current platform when you omit the `-r` parameter.
 
-## Framework-dependent executables (FDE)
+If the app uses a NuGet package that has platform-specific implementations, dependencies for all platforms are copied to the publish folder along with the app.
 
-Starting with .NET Core 2.2, you can deploy your app as an FDE, along with any required third-party dependencies. Your app will use the version of .NET Core that's installed on the target system.
+A runtime-dependent app can be created with the [`dotnet publish`](../tools/dotnet-publish.md) command. With .NET Core 3.0 SDK or higher, both a *dll* and executable file will be created.
```

Comment:
The second sentence repeats what was said above in the sentence starting with "Starting with .NET Core 3.0 SDK..."